tests/unit/components/slotMapper/reagentTransferSlotMapper.spec.tsx

```diff
@@ -0,0 +1,35 @@
+import { render, cleanup, screen } from '@testing-library/react';
+import ReagentTransferSlotMapper from '../../../../src/components/slotMapper/ReagentTransferSlotMapper';
+import { getById } from '../../generic/utilities';
+import { createLabware } from '../../../../src/mocks/handlers/labwareHandlers';
+import { enableMapSet } from 'immer';
+
+enableMapSet();
+afterEach(() => {
+  cleanup();
+});
+const srcLabware = createLabware('123456789123456789012345');
+const destLabware = createLabware('STAN-5111');
+
+describe('ReagentTransferSlotMapper.tsx', () => {
+  it('displays the parent div elements for source and destination', () => {
+    const reagentSlotMapper = render(
+      <ReagentTransferSlotMapper initialSourceLabware={undefined} initialDestLabware={undefined} />
+    );
+
+    //Displays div element for source labware
+    const sourceLabwareElem = getById(reagentSlotMapper.container, 'sourceLabwares');
```

Comment:
You could add a `data-testid=sourceLabwares` to the reagentTransferSlotMapper components to use as a unique identifier for the tests. Then get the component by its testId in the tests like c`onst sourceLabwareElem = screen.getByTestId('sourceLabwares')`